### PR TITLE
pull

### PR DESCRIPTION
**Pull request by aikongmeng**

### CODE REVIEW

File: src/com/activeandroid/DatabaseHelper.java [hunk 1]
```diff
@@ -213,7 +213,7 @@ private void executeSqlScript(SQLiteDatabase db, String file) {
 
 		    }
 
-		} catch (IOException e) {
+		} catch (Exception e) {
```
Comment (naXa777) [inline src/com/activeandroid/DatabaseHelper.java:216]: Why?

File: README.md [hunk 2]
```diff
@@ -16,14 +17,27 @@ Grab via Maven:
...
+			...
+			maven { url 'https://jitpack.io' }
+		}
+	}
+```
+ Add the dependency
+ ```groovy
+ 	dependencies { compile 'com.github.aikongmeng:ActiveAndroid:3.1'}
+```
+fix  跨版本更新sql修改表字段时，提示重复添加字段。
```
Comment (naXa777) [inline README.md:35]: I can't understand this.

File: README.md [hunk 3]
```diff
@@ -16,14 +17,27 @@ Grab via Maven:
...
+allprojects {
+		repositories {
+			...
+			maven { url 'https://jitpack.io' }
+		}
+	}
+```
+ Add the dependency
+ ```groovy
+ 	dependencies { compile 'com.github.aikongmeng:ActiveAndroid:3.1'}
```
Comment (naXa777) [inline README.md:33]: this is a wrong dependency! Revert back to `com.michaelpardo:activeandroid`

File: README.md [hunk 4]
```diff
@@ -16,14 +17,27 @@ Grab via Maven:
...
-    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
-}
+Add it in your root build.gradle at the end of repositories:
+
 
-compile 'com.michaelpardo:activeandroid:3.1.0-SNAPSHOT'
+```groovy
+allprojects {
+		repositories {
+			...
```
Comment (naXa777) [inline README.md:26]: `...` is a syntax error in Groovy.

File: README.md [hunk 5]
```diff
@@ -16,14 +17,27 @@ Grab via Maven:
...
-repositories {
-    mavenCentral()
-    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
-}
+Add it in your root build.gradle at the end of repositories:
+
 
-compile 'com.michaelpardo:activeandroid:3.1.0-SNAPSHOT'
+```groovy
+allprojects {
```
Comment (naXa777) [inline README.md:24]: Why `allprojects`? What if somebody wants to configure this for a specific module? or subprojects?

File: README.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 [![Build Status](https://travis-ci.org/pardom/ActiveAndroid.png?branch=master)](https://travis-ci.org/pardom/ActiveAndroid) [![Stories in Ready](https://badge.waffle.io/pardom/ActiveAndroid.png)](http://waffle.io/pardom/ActiveAndroid)  
-# ActiveAndroid
+# ActiveAndroid [![](https://jitpack.io/v/aikongmeng/ActiveAndroid.svg)](https://jitpack.io/#aikongmeng/ActiveAndroid)
```
Comment (naXa777) [inline README.md:2]: why `aikonmeng`? this repository belongs to @pardom-zz...

**Review by naXa777 — CHANGES_REQUESTED** [review]
Looks like spam.

File: README.md (hunk 6 above)
Comment (aikongmeng) [reply README.md]: I thought the author was not maintaining the project。
Replace this with `aikongmeng` ,  just for myself use.
Sorry,I wanted to Pull ,Push was Error operation.

Comment (aikongmeng) [reply README.md:2]: > why `aikonmeng`? this repository belongs to @pardom-zz...

I thought the author was not maintaining the project。
Replace this with aikongmeng , just for myself use.
Sorry,I wanted to Pull ,Push was Error operation.

File: src/com/activeandroid/DatabaseHelper.java (hunk 1 above)
Comment (aikongmeng) [reply src/com/activeandroid/DatabaseHelper.java:216]: > Why?

I wanted to solve  this err `duplicate column name` , you can see issues #542

**Pull request closed without merging**